src/main/java/frc/utils/interpolator/DiscritizationInterpolator.java

```diff
@@ -0,0 +1,34 @@
+package frc.utils.interpolator;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class DiscritizationInterpolator extends DoubleBilinearInterpolation {
+
+	public static final double DEFAULT_FUDGE_FACTOR = 1;
+	public DiscritizationInterpolator() {
+		super();
+	}
+
+	@SafeVarargs
+	public DiscritizationInterpolator(Pair<Translation2d, Double>... knownPoints) {
+		super(knownPoints);
+	}
+
+	@SuppressWarnings("finally")
+	@Override
+	public double getInterpolatedValue(Translation2d query) {
+		double returnValue = Double.NaN;
+		try {
+			returnValue = super.getInterpolatedValue(query);
+		} catch (Exception e) {
+			returnValue = Double.NaN;
+			System.out.println(e);
```

Comment:
print?

Reply:
Why later the alert manager in already in core